andela/ah-frontend-dojo
# #165273540:  Users should be able to follow each other

**What does this PR do?**

- Users should be able to follow and unfollow each other

**Description of Task to be completed?**

- Authors should follow each other

- Authors should unfollow each other

- Authors should be able to view his/her followers

- Authors should be able to view his/her followings

**How should this be manually tested?**

- Check out `git checkout ft-follow-unfollow-165273540`

- `npm install && npm run dev`

- log in with your account credentials

- Append an existing users username to the route to navigate to a users profile
e.g. https://localhost:8080/profile/{username}

**What are the relevant pivotal tracker stories?**

- [#165273540](https://www.pivotaltracker.com/story/show/165273540)

**Screenshots (if appropriate)**

<img width="1365" alt="Screenshot 2019-06-21 at 10 13 54" src="https://user-images.githubusercontent.com/38062858/59904880-57cc4000-940d-11e9-9990-295277454598.png">
